src/internal/operations/backup.go

```diff
@@ -338,6 +349,44 @@ func (op *BackupOperation) do(
 	return deets, nil
 }
 
+func Precheck(
```

Comment:
Don't think this needs to be exported.  Also, please add: a header comment explaining the high level purpose and behavior of this func.  

Name nit:
```suggestion
func CheckIsRunnable(
```